python/torch_mlir/__init__.py

```diff
@@ -43,6 +43,8 @@ class OutputType(Enum):
     # Raw output of the JIT IR importer. This is not expected to be useful
     # for end-users, but can be convenient for development or reporting bugs.
     RAW = 3
+    # This output type consists of `mhlo` dialect ops.
```

Comment:
Nit: please put blank link above this comment. Also, add "It can be thought of
    # as taking the `TORCH` output type and lowering it to MHLO."